### PR TITLE
zmap: fix dependencies

### PR DESCRIPTION
**Pull request by Grimler91**
The libraries are runtime dependencies.

### CODE REVIEW

File: root-packages/zmap/build.sh [hunk 1]
```diff
@@ -3,9 +3,10 @@ TERMUX_PKG_DESCRIPTION="A fast single packet network scanner designed for Intern
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="3.0.0-beta1"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/zmap/zmap/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=c97c452930ad6af95ad9ad79e66fea742f904b8e102c185f1001822cc5b5ceb3
-TERMUX_PKG_BUILD_DEPENDS="json-c, libgmp, gengetopt, libpcap, libunistring"
+TERMUX_PKG_DEPENDS="json-c, libgmp, libiconv, libpcap, libunistring"
```
Comment (xtkoba) [inline root-packages/zmap/build.sh:9]: Is `libiconv` also a runtime dependency?

Comment (Grimler91) [reply root-packages/zmap/build.sh:9]: Current package links against it:

```
$ ldd $PREFIX/bin/zmap
[...]
libiconv.so => /data/data/com.termux/files/usr/lib/libiconv.so
[...]
```

Comment (Grimler91) [reply root-packages/zmap/build.sh:9]: Or no, you are right, I guess iconv is a dependency of one of the other ones, it's not listed by objdump:

```
$ objdump -x $PREFIX/bin/zmap | grep NEEDED
  NEEDED          libpcap.so.1
  NEEDED          libgmp.so
  NEEDED          libunistring.so
  NEEDED          libjson-c.so
  NEEDED          libc.so
```

Comment (xtkoba) [reply root-packages/zmap/build.sh:9]: That is not a direct dependency. Probably a dependency of `libunistring` instead. Indirect dependencies should not be specified in `TERMUX_PKG_DEPENDS`.

**Pull request merged**